.env.sample

```diff
@@ -22,6 +22,11 @@
 # The URL for an external help resource; defaults to an empty string
 # HELP_URL=https://github.com/tl-its-umich-edu/canvas-app-explorer
 
+# (optional) The token for accessing the /watchman URL; leaving it blank means the route is unprotected
```

Comment:
When I was trying to make a call with authorization header via curl or postman, I received a forbidden error.  This is how I made my query, not sure I did this right

`curl -X GET -H "Authorization:  token=\”xyh\”” https://dev-cae/watchman`

Reply:
I got this to work. Looks like you would form it like:
```
curl -X GET -H "Authorization: {your custom token name} Token=\”xyh\”” https://dev-cae/watchman
```
The environment variable value goes after `Authorization: `. You have to use the `Token=` form after that.

Reply:
I think this will work since AWS outage the app is not reachable.